modules/search/search.py

```diff
@@ -0,0 +1,38 @@
+import math
+import numpy as np
+from numpy import sin, cos
+
```

Comment:
Do you intend for search.py to be a module? I think this should be a module since its processing logic, rather than process logic (i.e this is turning heading and GPS into a bearing, then the search **program** in the main process can take the bearing and send it off to the command module. 

Since this should be a module, put this inside a class, define an init function, make alt search a method of said class. Don't make any parameters of the function class member variables, since they're constantly regenerated they'll come in through a pipeline.